roadmap.md

```diff
@@ -3,44 +3,69 @@ layout: default
 ---
 # Roadmap
 
-## WebAssembly Consensus
+In November 2017, WebAssembly CG members representing four browsers, Chrome, Edge, Firefox, and WebKit, have reached consensus that the design of the initial (MVP) WebAssembly API and binary format is complete to the extent that no further design work is possible without implementation experience and significant usage.
```

Comment:
s/have reached/reached/

Reply:
This was the original text, but I agree reached is better here.

Reply:
FWIW you can use Github suggestions, they make merging fixes easier :)

```suggestion
In November 2017, WebAssembly CG members representing four browsers, Chrome, Edge, Firefox, and WebKit, reached consensus that the design of the initial (MVP) WebAssembly API and binary format is complete to the extent that no further design work is possible without implementation experience and significant usage.
```